readme.md

```diff
@@ -113,19 +113,236 @@ can be found in our Kubernetes configuration https://github.com/zalando-incubato
 Skipper's Godoc page, https://godoc.org/github.com/zalando/skipper, includes detailed information on these
 topics:
 
-- The Routing Mechanism
+- The [Routing](https://godoc.org/github.com/zalando/skipper/routing) Mechanism
 - Matching Requests
-- Filters - Augmenting Requests
+- [Filters](https://godoc.org/github.com/zalando/skipper/filters) - Augmenting Requests
 - Service Backends
 - Route Definitions
-- Data Sources: Kubernetes
-- Circuit Breakers
+- Data Sources: [ETCD](https://godoc.org/github.com/zalando/skipper/etcd), [Inkeeper API](https://godoc.org/github.com/zalando/skipper/innkeeper), [Kubernetes](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes), [Route string](https://godoc.org/github.com/zalando/skipper/dataclients/routestring)
```

Comment:
i think the etcd project likes 'etcd' in all lowercase. At least that's how it appears in all of their documentation.